pages/home_page.py

```diff
@@ -5,16 +5,28 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
 from pages.base import Base
 
 
 class Home(Base):
 
+    _groups_link_locator = (By.PARTIAL_LINK_TEXT, 'groups')
+    _functional_areas_link_locator = (By.PARTIAL_LINK_TEXT, 'functional areas')
```

Comment:
Good point. I didn't consider l10n here. There isn't a class/id that is super specific which is why I didn't use it initially (I saw other places using partial text and followed suit). Let me know what you think of section.class > a. Otherwise I was thinking I could use the href text (something like a[href*="functional-areas"]).